scripts/ci/ci_before_install.sh

```diff
@@ -15,25 +15,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+set -x
```

Comment:
Debug or left in on purpose?

Reply:
left it on purpose. ci_before install is folded in Travis so it's ok to get more information and sometimes it's easier to debug it this way